operations/flags.go

```diff
@@ -298,6 +299,28 @@ func addUncommittedChangesFlag(flags ...cli.Flag) []cli.Flag {
 	})
 }
 
+func addReuseFlags(flags ...cli.Flag) []cli.Flag {
```

Comment:
I think the help output might be easier to read quickly if it looked something like this: 
![image](https://user-images.githubusercontent.com/13104717/203602808-566ea0a3-d9a0-452c-afed-3a455767a69b.png)

Reply:
This isn't something we can control easily since it's handled by the library: https://github.com/urfave/cli/blob/main/help.go#L243

Reply:
I'm referring to appending either "repeat latest: " or "repeat specific: " to `message`

Reply:
ohh i see, thanks. sure